src/transform/namer.ts

```diff
@@ -45,7 +45,7 @@ export async function namer(session: Session<CodeModel>) {
     // this looks like it already has data for this model.
     // send back an error
     session.error('bad flavor', ['go:1000', 'already-processed'], model.language.go);
-    throw Error('Go Namer Failed');
```

Comment:
Not saying to do this now, but could this be improved?

Reply:
I think this is one of those "this should never happen but just in case" type of error condition, i.e. if we hit this then we broke something elsewhere (Garrett had added this in the template project).